content/case_histories/hoop_region_norway/properties.rst

```diff
@@ -0,0 +1,100 @@
+.. _hoop_region_norway_properties:
+
+
+
+.. |lo| unicode:: 0xF8
+   :ltrim:
+.. |o| unicode:: 0xF8
+   :trim:
+.. |ao| unicode:: 0xE5
+   :trim:
+
+
+Properties
+==========
+
+Detailed petrophysical evaluation and rock physics analysis was carried out for the Wisting Central and Wisting Alternative wells. Lithology and fluid sensitivity of the elastic attributes were addressed in the rock physics analysis and the main results obtained are illustrated in :numref:`fig_poisson_acoustic_impedance`. This figure shows a cross-plot of the elastic attributes Poisson’s ratio versus acoustic impedance, color coded by four different fluid substituted saturations: 100% water, 80% oil, 80% gas, 20% gas (fizz saturation). Also shown is the in-situ saturation for the St |lo| Fm., which is a clean, blocky shallow-marine shoreface sandstone, as well as for the underlying Nordmela Fm, which is a coarsening upwards shaly sand.
+
+
+Two important observations can be extracted from these plots. The first is the well-known inability of the seismic data alone to discriminate between commercial and non-commercial gas saturation: The 20% and 80% gas cases plot on top of each other, which is not the case for the combination of elastic and electric attributes (see :numref:`fig_resistivity_acoustic_impedance`). The second observation is the unexpected high Poisson’s ratio (or higher Vp/Vs) in the St |lo| Fm. compared to that observed in Nordmela Fm. for the same fluid saturation. It is worthwhile mentioning that the same observation was found in both the Wisting Central and Wisting Alternative wells, which suggests this unusual response is not a measurement problem. Moreover, the fact that the AVO inversion produced a Poisson’s ratio solution that also honors these slightly unusual well log observations of Poisson’s ratio for the St |lo| and Nordmela Fms. (:numref:`fig_inversion_initial_pwave_poisson`) validates these measurements.
+
+
+.. figure:: images/poisson_vs_acoustic_impedance.png
+    :align: center
+    :figwidth: 80%
+    :name: fig_poisson_acoustic_impedance
+
+a) *For the St |lo| Fm., multi-well cross-plot of Poisson’s ratio versus acoustic impedance color coded by fluid saturation with background colored by volume of clay for the full well.*
```

Comment:
Can you please indent this so it is treated as a figure caption? That way it will also be numbered, which it currently isn't. Then you can remove the Italics command, and that should help the |lo| render properly (currently it is not working inside of the italics command)